app/features/add-payment-address-form.tsx

```diff
@@ -13,19 +13,32 @@ function SelectLabel({ token }: { token: Token }) {
   );
 }
 
-export function AddPaymentAddressForm({ tokens }: { tokens: Token[] }) {
+export function AddPaymentAddressForm({
+  tokens,
+  setSelectedNetwork,
+  setSelectedAddress,
+}: {
+  tokens: Token[];
+  setSelectedNetwork: any;
+  setSelectedAddress: any;
```

Comment:
For future reference we should never use `any`. In this case we know we need a function so `setSelectedNetwork: (e. React.FormEvent<HTMLInputElement>) => void` is probably the type we want, or something like that.

But it looks like these props aren't used anyways?

Reply:
oh good catch forgot to update these types, was testing if they work. They are used. They are used onChange in this form so that the parent component in addresses.tsx has the state to check for valid addresses

Reply:
having trouble typing this tbh

Reply:
![Screen Shot 2022-12-19 at 5 40 03 PM](https://user-images.githubusercontent.com/43100701/208540568-1e4119ed-2e7b-44f3-8e0e-66795b33cbc5.png)

Reply:
oh jk I think I did it